capstone/src/main/webapp/youtubeScript.js

```diff
@@ -78,21 +65,20 @@ function loadChartsApi(toxicityData) {
   google.charts.setOnLoadCallback(function() {drawBarChart(toxicityData);}); 
 }
 
-/** Draws a Google BarChart from a Perspective JSON. */
+/** Draws a Google BarChart from a map. */
 function drawBarChart(toxicityData) {
   document.getElementById('chart-container').innerHTML = '';
   const data = google.visualization.arrayToDataTable([[{label: 'Attribute'}, {label: 'Score', type: 'number'}, {role: "style"}]]);
-
-  Object.keys(toxicityData.attributeScores).forEach((attribute) => {
+  for (let [attribute, attributeScoresAvg] of toxicityData) {
     var color = '#6B8E23'; // Green
-    const score = toxicityData.attributeScores[attribute].summaryScore.value;
+    const score = attributeScoresAvg;
     if (score >= 0.8) {
       color = '#DC143C'; // Red
     } else if (score >= 0.2) {
       color = '#ffd800'; // Yellow
     }
-    data.addRow([attribute, score, color]);
-  });
+    data.addRow([attribute , score, color]);
```

Comment:
Nit: let's remove the space that was inserted after `attribute`

Reply:
addressed in "address nits"